### PR TITLE
feat: wishlist

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added wishlist page

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+        <div class="input-container">
+            <h1>Your wishlist</h1>
+        
+            <div class="card-container">
+              <div class="card">
+                <img class="card-img" src="/assets/images/men-shirt.jpg" alt="">
+                <span class="bdg-card bdg-icon red">New</span>
+                <h3>Men shirt</h3>
+                <h4 class="t-cntr">Price: Rs.400</h4>
```
Comment (JuvitaSaini09) [inline pages/wishlist.html:59]: you can use &#8377 instead of Rs. ,justa suggestion

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                <p class="t-cntr">This is men's shirt</p>
+                <div class="card-btn-container">
+                <button class="btn green">Cart</button>
+                <button class="btn outline-red">Remove</button>
+                </div>
+             </div>
+
+             <div class="card">
+                <img class="card-img" src="/assets/images/saree.jpg" alt="">
```
Comment (JuvitaSaini09) [inline pages/wishlist.html]: empty alt tag

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks good to merge

**Review by palash23jain — APPROVED** [review]
you can prefer using semantic tags in HTML where ever possible. For now, this looks good to go

**Pull request merged**